include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -256,7 +280,9 @@ struct layout_stride {
 #else
       : __base_t(__base_t{__member_pair_t(
```

Comment:
Another instance of repeated `__base_t`.

Reply:
I will look at that later, revamping layout_stride itself